app/javascript/components/map/mixins/mixin-add-layers.js

```diff
@@ -1,21 +1,29 @@
 import { getFirstDataLayerId } from '../helpers/map-helpers'
 
-const addImagePaintOptions = (map, options, layer) => {
-  addFillPaintOptions(options, layer)
-  const patternId = 'pattern' + layer.id
-
-  options['paint']['fill-pattern'] = patternId
-
+const addImageLayer = (map, options, layer, nextLayer) => {
   map.loadImage(
     layer.image,
     (err, image) => {
-      if (err) { throw err }
-      
-      map.addImage(patternId, image)
+      if (err) { 
+        console.log(`Image ${layer.image} not found. Falling back to solid fill.`)
```

Comment:
Should this be console.error or console.warn?

Reply:
Hmmm potentially, yes... I think warn would be best as it fails gracefully